azurerm/internal/services/securitycenter/tests/resource_arm_security_center_automation_test.go

```diff
@@ -605,6 +605,11 @@ resource "azurerm_security_center_automation" "test" {
       }
     }
   }
+
+  description = "Security Center Automation Acc test"
+  tags = {
+    Env = "Test"
+  }
```

Comment:
one of the other Security Center API's doesn't support updating tags - can we update this test to confirm this?

Reply:
done